package.json

```diff
@@ -62,8 +60,10 @@
     "@babel/eslint-parser": "^7.13.10",
     "@babel/plugin-proposal-class-properties": "^7.13.0",
     "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-modules-commonjs": "^7.13.8",
```

Comment:
Shouldn't those plugins now be now be pulled in automatically through the `@nextcloud/babel-config` package?

Reply:
Nope
with npm6 peers deps are not automatically pulled
BUT, npm7 put back this behaviour.

We just need to talk about that migration and potential loopholes.